### ISSUE TITLE
Allow typing dynamics from the keyboard

### ISSUE THREAD

**Issue by iainhallam** [musescore/MuseScore#21263]
### Your idea

Have an "add text: dynamics" command that can have a keyboard shortcut assigned (Ctrl+D is available in the default map) that parses the dynamic at the end of editing to set the playback.

### Problem to be solved

The only way to enter a dynamic from the keyboard is through the accessibility interface. Being a general interface, this takes much longer than a dedicated shortcut could. Rather than taking up extra shortcuts for each separate dynamic, a simple way to start editing a text object with the dynamic type would be economical and quick.

### Prior art

Sibelius includes dynamics in its expression text, and uses a playback dictionary to find words that should affect playback, which makes it very powerful, and quick to use. They also have a right-click menu while editing text to add quick snippets appropriate to the type of text being entered.

**Comment by Tadaboody** [comment]
Is this issue still being worked on? if not I'd love to pick it up. 
I'll also say that in the handbook there already are keyboard shortcuts for dynamics but they seem not to work for me (OS: macOS 12.7, Arch.: x86_64, MuseScore Studio version (64-bit): 4.3.0-241231433, revision: github-musescore-musescore-5f36e74)

**Comment by MarcSabatella** [comment]
Yes, there is work being done on this as part of the Google Summer of Code - see https://musescore.org/en/user/5201410/blog/2024/05/28/gsoc-2024-dynamics-popup

**Comment by oktophonie** [comment]
Closing as implemented in 4.5
